linetools/isgm/utils.py

```diff
@@ -407,6 +426,7 @@ def iontable_from_components(components, ztbl=None, NHI_obj=None):
     -------
     iontbl : Table
     """
+    warnings.warn("It is likely this method will be Deprecated", DeprecationWarning)
```

Comment:
Fine by me, do not think I have used this much.

Reply:
k